tutorial/advanced_integration.md

```diff
@@ -96,34 +97,36 @@ In our example below, we disable the 'Order' button when the agent is in control
 
 ``` html
 <script>
-var settings = {
-widget_key:'**your widget key here**',
-cookie_transfer_enabled: true,
-cookie_transfer_proxying: false
-};
-
-window.addEventListener('DOMContentLoaded', function() {
-  Surfly.init(settings, function(init) {
-    if (init.success) {
-      var sess;
-      if(!Surfly.currentSession){
-		sess = Surfly.session();
-      } else {
-		sess = Surfly.currentSession;
-      }
-      sess.on('control', function(session, event) {
-            var element = document.getElementById("order_button");
-    		if (event.to==0) {
-       			element.disabled = false;
-       			element.style.backgroundColor = "#87cefa";
-            } else {
-       			element.disabled = true;
-       			element.style.backgroundColor = "#e6fff2";
-    		}
-      })
-	}
+  var settings = {
+    widget_key:'**your widget key here**',
+    cookie_transfer_enabled: true,
+    cookie_transfer_proxying: false
+  };
+
+  window.addEventListener('DOMContentLoaded', function() {
+    Surfly.init(settings, function(init) {
+      if (init.success) {
+        var sess;
+        if(!Surfly.currentSession){
+  		    sess = Surfly.session();
```

Comment:
There are still some tabs in this file, it looks weird, let's replace all tabs with spaces